packages/grid/_modules_/grid/models/gridOptions.tsx

```diff
@@ -82,6 +82,11 @@ export interface GridOptions {
    * @default false
    */
   disableDensitySelector?: boolean;
+  /**
+   * If `true`, CSV export option is disabled.
+   * @default false
+   */
+  disableCsvExport?: boolean;
```

Comment:
The prop seems to only impact the toolbar. I think that we should encourage developers to create their own toolbar if they don't want the default. I would propose we remove `disableDensitySelector` too as it's toolbar only.

@mui-org/x thoughts?

Reply:
The reason why I added it is because when in the future you have CSV and PDF export options for example you could disable one of them. Also, it will help us control what is available and what no on the open-source version of the data grid.

I agree - for now it may not be useful but I think of it as more of an approach to the problem. Another option is to remove it for the CSV since it is a free feature but add one for the Exel of PDF when we pick them up.

Reply:
> in the future you have CSV and PDF export options for example you could disable one of them

Did you consider using the export toolbar component to host these props? Does it need to be in the root component?

Reply:
Yes, I was thinking to build on this component later.  By root component, I guess you mean the grid root component?

Reply:
> By "root component", I guess you mean the grid root component?

Yes

Reply:
Ok, wanted to be sure - no I don't think it needs to be in the root. If we have props to disable different export types we can force them on the `DataGrid`

Reply:
> I was thinking to build on this component later.

What's the motivation for not doing it now?

Reply:
What I meant by "building on" was to extend it with more options like PDf export and etc. I don't think we need to provide the same level of extendibility as we do for the toolbar in this case. I think a simple "alow all options by default" and "hide whatever you don't need" is ok. If neither of those options works for some specific use cases devs can always do their own implementation using the API we provide.

Reply:
Ok, I think we are not covering the same topic. My concern is that the prop is on the `DataGrid` component. I don't understand why it's not on the `GridToolbarExport` component or even `GridToolbar`.

Reply:
I think that at this stage it is useless to have the disable csv export option, this is because it is already opt in within the toolbar. We have the toolbar composability so it seems redundant to have the option as well. You could just add a prop on the component

Reply:
Ok, my bad, I didn't understand the point correctly, I'm convinced, will remove it. If at a later stage we find the demand for such functionality we can revise it.